src/test/rustdoc/crate-title.rs

```diff
@@ -0,0 +1,3 @@
+#![crate_name = "foo"]
+
+// @has foo/index.html '//head/title' 'foo - Rust'
```

Comment:
I would put this in item-title too.

Reply:
I've merged all of the `<title>` tests into one test.